packages/cli/src/commands/model/list.ts

```diff
@@ -12,17 +13,24 @@ type PartialModelDefinition = {
 }
 
 type ModelListFlags = QueryCommandFlags & {
+  'indexer-url': string
   table?: boolean
 }
 
-export default class ModelList extends Command<ModelListFlags> {
+export default class ModelList extends BaseCommand<ModelListFlags> {
   fetchedFields: Array<PartialModelDefinition> = []
   lastLoadedPageInfo: PageInfo | null = null
 
   static description = 'load a list of models with pagination'
 
   static flags = {
-    ...Command.flags,
+    ...BaseCommand.flags,
+    'indexer-url': Flags.string({
+      default: 'https://ceramic-private-clay.3boxlabs.com',
+      char: 'i',
+      description: 'URL of a Ceramic API that indexes all models',
+      env: 'INDEXER_URL',
```

Comment:
I don't think we need to add an environment variable here considering this is the only command using the indexer URL?